### ISSUE TITLE
No results when running clinic doctor for HAPI server

### ISSUE THREAD

**Issue by aschekatihin** [nearform/node-clinic#129]
## Expected Behavior
An html report generated

## Current Behavior
No html files created, clinic creates folder `1888.clinic-doctor` containing file `1888.clinic-doctor-processstat` and `1888.clinic-doctor-systeminfo` and that's all.

## Steps to Reproduce (for bugs)
1. clinic doctor -- node build/src/server.js
2. wait for server to initialize
3. press ctrl-c

## Sample upload
https://upload.clinicjs.org/public/97f8d6ff2dadfe11ab3288e48383826a800fc51a31a7339690a768d51111c960/1888.clinic-doctor.html
But as no html actually generated this url doesn't works anyway.

## Environment
* Clinic version: 3.0.0
* Node version: 8.9.4
* Operating system and version: Windows 10.0.17134.619

Is there some verbose mode or logs to understand what goes wrong?

Also tried to install clinic locally (without -g parameter) - it creates incomplete html without body with header section only. But there is no error messages.

It looks like clinic is interrupted by ctrl-c instead of (before) server.

**Comment by mcollina** [comment]
Are you running any load at all on the server?

Can you please include an example server to reproduce?

**Comment by aschekatihin** [comment]
Yes, of course I did a few request to the server.
I can't provide my current server as a sample and will need some time to prepare separate sample.
So if there are logs or some other diagnistic tool it would be a bit easier to understand what's going on.

**Comment by chaoyangnz** [comment]
the same issue

**Comment by thearegee** [comment]
I'm having the same issue with my fastify server. 

Running this command to start it:
```
$ ./node_modules/clinic/bin.js doctor -- node --max-http-header-size=25000 lib/index.js
```
I start load test using our internal tool in another tab, once complete I close the server with `ctrl+c` and it generates the `11810.clinic-doctor` folder structure but not the html file. I can see the server is getting load.

However if I use the inbuilt autocannon command:

```
./node_modules/clinic/bin.js doctor --on-port './node_modules/autocannon/autocannon.js localhost:1337' -- node lib/index.js
```
This works and generates a HTML file with the results. Any ideas why that might be @mcollina ?

**Comment by goto-bus-stop** [comment]
should be fixed in `clinic@4.0.3` by https://github.com/nearform/node-clinic/pull/150

**Comment by Dhruv-Garg79** [comment]
Hey I am facing the same issue in fastify using latest clinic@13.0.0
